src/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.csproj

```diff
@@ -22,7 +22,7 @@
     <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildVersion)" />
-    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonSoftVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonSoftLegacyVersion)" />
```

Comment:
This needs to stay at older version till we start using newer msbuild (CLI) that supports the `10.0.1` version.